event_routing_backends/processors/tests/transformers_test_mixin.py

```diff
@@ -108,6 +108,14 @@ def test_event_transformer(self, event_filename, mocked_uuid):
         with open(expected_event_file_path, encoding='utf-8') as expected:
             expected_event = json.loads(expected.read())
 
-            actual_transformed_event = self.registry.get_transformer(original_event).transform()
+            try:
+                actual_transformed_event = self.registry.get_transformer(original_event).transform()
+                self.compare_events(actual_transformed_event, expected_event)
+            except Exception as e:
+                with open("actual_transformed_event.json", "w") as actual_transformed_event_file:
+                    actual_transformed_event_file.write(actual_transformed_event.to_json())
 
-            self.compare_events(actual_transformed_event, expected_event)
+                with open("expected_event.json", "w") as expected_event_file:
+                    json.dump(expected_event, expected_event_file, indent=4)
+
+                raise e
```

Comment:
This is to render the expected fixture to avoid that manual work. It only run on error, so those can be easily found

Reply:
Can you exclude all of these lines from test coverage? Seems reasonable to do since they should only be triggered in a test failure.

Reply:
Done